packages/notifications-system/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "message",
+  "name": "@booking-services/notifications-system",
```

Comment:
Yup, it was me, I did it because I wasn't able to compile the project, it appears that it had the same name of another package of another module, so I changed it.